src/drawing/DrawingList.tsx

```diff
@@ -40,6 +41,17 @@ export function DrawingList({ activeId = null, annotations, className, onSelect
     });
     useOutsideEvent('mouseup', rootElRef, (): void => setIsListening(true));
 
+    React.useEffect(() => {
+        const { current: rootEl } = rootElRef;
+        if (!rootEl) {
+            return;
+        }
+        setRootDimensions({
+            height: rootEl.clientHeight,
+            width: rootEl.clientWidth,
+        });
+    }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
     return (
         <svg
```

Comment:
Should we create a `DrawingSVG` component to encapsulate some of these attributes that will be shared across create & render drawings? `viewBox` & `preserveAsepctRatio`. Possibly even the filter definition?